src/schema/me/__tests__/bidder_position_mutation.test.js

```diff
@@ -1,33 +1,126 @@
 import { runAuthenticatedQuery } from "test/utils"
+import config from "config"
 
-describe("Bidder position mutation", () => {
-  const createBidderPosition = {
-      suggested_next_bid_cents: 110000,
-  }
-
-  const query = `
+const query = `
   mutation {
     createBidderPosition(input: {
       artwork_id: "daryl-daniels-free-thyself-1"
       max_bid_amount_cents: 100000
-      sale_id: "one-year-of-resistance-benefit-auction-2018"
+      sale_id: "sixteen-year-of-resistance-benefit-auction-2032"
     }) {
-      position {
-        suggested_next_bid_cents
+      result {
+        status
+        position {
+          suggested_next_bid_cents
+        }
+        message_header
+        message_description_md
       }
     }
   }
   `
 
-  const rootValue = {
-    createBidderPositionLoader: sinon.stub().returns(
-      Promise.resolve(createBidderPosition)
-    ),
+const errorMEssageTemplate = `{ Error: https://stagingapi.artsy.net/api/v1/me/bidder_position?
+        artwork_id=mohammed-qasim-ashfaq-surrealism&max_bid_amount_cents=3700000&
+        sale_id=bidding-test - `
+
+describe("Bidder position mutation", () => {
+  const createBidderPosition = {
+    suggested_next_bid_cents: 110000,
   }
 
-  it("creates a bidder position", async () => {
-    return runAuthenticatedQuery(query, rootValue).then(data => {
-      expect(data.createBidderPosition.position.suggested_next_bid_cents).toEqual(110000)
+  describe("success", () => {
+    it("creates a bidder position", async () => {
+      const rootValue = {
+        createBidderPositionLoader: sinon.stub().returns(
+          Promise.resolve(createBidderPosition)
+        ),
+      }
+      return runAuthenticatedQuery(query, rootValue).then(data => {
```

Comment:
Since you're in an `async` function already, it would be better to do:

```
  describe("success", () => {
    it("creates a bidder position", async () => {
      const rootValue = {
        createBidderPositionLoader: sinon.stub().returns(
          Promise.resolve(createBidderPosition)
        ),
      }
      const data = runAuthenticatedQuery(query, rootValue)
      expect(data.createBidderPosition.result.position.suggested_next_bid_cents).toEqual(110000)
      expect(data.createBidderPosition.result.message_header).toBeNull()
      expect(data.createBidderPosition.result.message_description_md).toBeNull()
    })
  })
```

In a world with `async`/`await`, I'd almost never use `.then` explicitly. The only time I can think of where it's the way to go is if you need two paths of execution to happen concurrently after a promise resolves, which should be really rare.